src/notifications/warning-message.ts

```diff
@@ -49,5 +49,6 @@ export default {
     deprecatedAPI:                      '{API} is deprecated and will be removed in the next major release. Use {replacement} instead.',
     unawaitedMethodWithAssertion:       "An asynchronous method that you do not await includes an assertion. Inspect that method's execution chain and add the 'await' keyword where necessary.",
     multipleConfigurationFilesFound:    'There are multiple configuration files found, TestCafe will only use one. The file "{path}" will be used.\nThe priority order is as follows:\n{priorityList}',
```

Comment:
we should probably fix this one as well, while we're at it

```suggestion
    multipleConfigurationFilesFound:    'TestCafe detected more than one configuration file.\nTo prevent configuration conflicts, TestCafe will use the configuration file with the highest priority: {path}. Refer to the following article for more information: https://testcafe.io/documentation/402638/reference/configuration-file?search#configuration-file-priority',
```